localization/english/event_localization/wc_central_kalimdor_events_l_english.yml

```diff
@@ -0,0 +1,250 @@
+cen_kal.1055.a:0 "Prepare the ritual, Thornweavers..."
+cen_kal.1055.a.tt:0 "Your thornweavers will prepare a ritual to call forth large thorn covered vines in the county of [thorn_location.GetName]."
+cen_kal.1055.b:0 "Perhaps it's best to wait..."
+
+cen_kal.1056.desc:0 "You have summoned your head thornweaver - [SCOPE.sC('druid').GetName], a respected and skilled elder of your tribe. They have a serious and urgent matter to discuss with you. You have heard rumors of a county that is rich in the blood of Agamaggan and suitable for the thorn calling ritual. You ask your head thornweaver to confirm these rumors, and to investigate the county for you. They agree, and after some time, they have returned with bad news. The county that you mentioned is not worthy of being blessed by the holy thorn covered veins and hold no trace of the Agamaggan’s blood in the soil.\n\nThey apologize for disappointing you, and ask you to forgive them for their failure. They say that they have wasted your time and resources, and that they have let you down as your head thornweaver. Before leaving, [SCOPE.sC('druid').GetName] suggests that perhaps it is time to expand our realm to find new lands that might be more fitting for the ritual."
+cen_kal.1056.a:0 "We have to grow further to recieve Agamaggan's blessing..."
+
+cen_kal.1058.desc:0 "You arrive at the ritual site in [thorn_location.GetName], where the thornweavers are preparing to call forth the thorns of Agamaggan. You see different elder thornweavers gathered around a large pit, each holding a blood shard in their hands. They are chanting and swaying, invoking the power of the boar god and his blood. You spot [SCOPE.sC('druid').GetFirstName], the head of your thornweavers, standing near the pit, waiting for you.\n\n“[ROOT.Char.GetTitleAsNameNoTooltip], welcome to Agamaggan’s ritual, we have been waiting for you. <snort> I am glad you could join us for this sacred occasion. <snort>”\n\nAfter the completion of the ritual, you will be able to expand the thorns further into stronger fortifications if you desire. "What do you wish, [ROOT.Char.GetTitleAsNameNoTooltip]? Shall we call forth the great thorns of Agamaggan? <snort>""
+cen_kal.1058.a:0 "Agamaggan blesses us with his power!"
+cen_kal.1058.b:0 "Perhaps we should stop this..."
+cen_kal.1058.a.tt:0 "You will use up a total of 12 blood shards"
+
+quilboar_empire_name:0 "Agamaggar"
+cen_kal.1060.name:0 "Agamaggar Beckons"
+cen_kal.1060.desc:0 "It has been ages since the quilboar last roamed freely in the Barrens, the twin threats of the Centaur and the Tauren pushing us to the brinks of survival. Sometimes the tauren speak of peace, sometimes of war, and sometimes the centaur speak of honor, sometimes of blood, but always these enemies scheme, and always our holy lands seem to be the target.\n\nAfter today, no more. I have reclaimed the old domains of Agamaggan, and the thorn-standard of Agamaggar rises!"
```

Comment:
```suggestion
cen_kal.1060.desc:0 "It has been ages since the quilboar last roamed freely in the Barrens, the twin threats of the centaurs and the tauren pushing us to the brink of survival. Sometimes, the tauren speak of peace, sometimes of war, and sometimes the centaur speak of honor, sometimes of blood, but always these enemies scheme, and always our holy lands seem to be the target.\n\nAfter today, no more. I have reclaimed the old domains of Agamaggan, and the thorn-standard of Agamaggar rises!"
```